static/js/browser2/browser_page.tsx

```diff
@@ -36,12 +38,37 @@ interface BrowserPagePropType {
 
 export class BrowserPage extends React.Component<BrowserPagePropType> {
   render(): JSX.Element {
-    const pageTitle = this.getPageTitle();
     const arcDcid = this.getArcDcid();
     return (
       <>
-        <div className="node-about">{"About: " + pageTitle}</div>
-        <div className="node-type">{"type: " + this.props.nodeType}</div>
+        <div className="browser-page-header">
+          {this.props.pageDisplayType === PageDisplayType.PLACE_STAT_VAR && (
+            <div className="browser-header-title">
+              <span>Statistical Variable: </span>
```

Comment:
Not all text needs to be wrapped in a span -- e.g. you could leave out the span tags, keep the text and it should render the same.

Reply:
done